packages/easy-coding-standard/src/Console/Command/CheckMarkdownCommand.php

```diff
@@ -67,17 +68,24 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             throw new NoMarkdownFileException($message);
         }
 
-        $markdownFileInfo = new SmartFileInfo($markdownFile);
-        $fixedContent = $this->markdownPHPCodeFormatter->format($markdownFileInfo);
+        try {
+            $markdownFileInfo = new SmartFileInfo($markdownFile);
+            $fixedContent = $this->markdownPHPCodeFormatter->format($markdownFileInfo);
 
-        if ($markdownFileInfo->getContents() === $fixedContent) {
-            $successMessage = 'PHP code in Markdown already follow coding standard';
-        } else {
-            $this->smartFileSystem->dumpFile($markdownFile, (string) $fixedContent);
-            $successMessage = 'PHP code in Markdown has been fixed to follow coding standard';
+            if ($markdownFileInfo->getContents() === $fixedContent) {
+                $successMessage = 'PHP code in Markdown already follow coding standard';
+            } else {
+                $this->smartFileSystem->dumpFile($markdownFile, (string) $fixedContent);
+                $successMessage = 'PHP code in Markdown has been fixed to follow coding standard';
+            }
+
+            $this->easyCodingStandardStyle->success($successMessage);
+        } catch (Throwable $throwable) {
         }
 
-        $this->easyCodingStandardStyle->success($successMessage);
+        // ensure clean up php-code-* files that undeleted yet because of parse error
+        $mask = 'php-code-*';
+        array_map('unlink', (array) glob($mask));
```

Comment:
Why do you prefer this location over the formatter service?

Reply:
I've moved the handling to `AbstractPHPFormatter`

Reply:
Fixed.